javascript-timers/interval.js

```diff
@@ -0,0 +1,23 @@
+var $h1 = document.querySelector('h1');
+
+function countDown() {
+  if ($h1.textContent === '4') {
+    $h1.textContent = '3';
+    return;
+  }
+  if ($h1.textContent === '3') {
+    $h1.textContent = '2';
+    return;
+  }
+  if ($h1.textContent === '2') {
+    $h1.textContent = '1';
+    return;
+  }
+  if ($h1.textContent === '1') {
+    $h1.textContent = '~Earth Beeeeloooww Us~';
+    clearInterval(intervalID);
+
+  }
```

Comment:
You should make this a conditional block with `else if` statements instead of a bunch of individual conditionals.